tests/test_config_entries.py

```diff
@@ -1143,16 +1143,20 @@ async def test_update_entry_options_and_trigger_listener(
     """Test that we can update entry options and trigger listener."""
     entry = MockConfigEntry(domain="test", options={"first": True})
     entry.add_to_manager(manager)
+    update_listener_calls = []
 
     async def update_listener(hass, entry):
         """Test function."""
         assert entry.options == {"second": True}
+        update_listener_calls.append(None)
```

Comment:
Is this doing anything more than being a simple counter?

If so, maybe an integer counter or boolean flag would be equivalent?

Reply:
Yeah, that's right, but we use this pattern in quite a lot of tests already.